sdk/include/xdk/ntbasedef.h

```diff
@@ -787,8 +787,10 @@ extern "C++" { \
  #define RotateRight16 _rotr16
 #endif /* _M_AMD64 */
 
-/* C_ASSERT Definition */
-#define C_ASSERT(expr) extern char (*c_assert(void)) [(expr) ? 1 : -1]
+/* Static assert */
+#ifndef C_ASSERT
```

Comment:
Our headers are supposed to be compatible with Microsoft's, so I wouldn't recommend adding checks like this "just because" (you may end up with applications that compile fine with ours but not with MS's)

Reply:
Fixed.